jumpscale/packages/tfgrid_solutions/chats/cryptpad_deploy.py

```diff
@@ -234,6 +233,14 @@ def reservation(self):
         self.container_url_https = f"https://{self.domain}"
         self.container_url_http = f"http://{self.domain}"
 
+    @chatflow_step(title="Initializing", disable_previous=True)
+    def intializing(self):
+        self.md_show_update("Initializing your Cryptpad ...")
+        if not j.sals.nettools.wait_http_test(self.container_url_https, timeout=600):
+            self.stop("Failed to initialize Cryptpad, please contact support")
```

Comment:
can we provide more info the ip, node, farm, reservation id? or what data we will contact the support with? because at this point the deployment was successful but the backend server of the solution didn't reply